nilearn/input_data/nifti_spheres_masker.py

```diff
@@ -38,9 +38,9 @@ def _apply_mask_and_get_affinity(seeds, niimg, radius, allow_overlap,
 
         X = masking._apply_mask_fmri(niimg, mask_img)
     else:
-        #Adding  warning message 
+        # Adding warning message
```

Comment:
The code clearly tells us that a warning is being raised here and under what condition so the comment is unwarranted.